assets/js/base/components/payment-methods/style.scss

```diff
@@ -36,6 +36,7 @@
 			width: 50%;
 			> img {
 				width: 100%;
+				height: 48px;
```

Comment:
Should this be max-height? Are we setting an expected/standard dimensions for gateways to use?

Reply:
I _think_ we want to ensure that there's some consistency in styling here. Otherwise multiple express payment methods could risk being varying heights?